pydis_site/apps/content/resources/guides/python-guides/fix-ssl-certificate.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Fix SSL certificate error.
+description: A guide about how to fix the SSL certificate error.
+---
+**How to fix SSL Certificate issue on Windows**
```

Comment:
Perhaps instead of using bold characters, we can use subheadings?
```suggestion
#### How to fix SSL Certificate issue on Windows**
```

Reply:
Why use a nested header rather than just `#`?